doc/explanation/microcloud.md

```diff
@@ -61,6 +61,19 @@ MicroCloud will still be usable, but you will see some limitations:
   As a result of this, network forwarding works at a basic level only, and external addresses must be forwarded to a specific cluster member and don't fail over.
 - There is no support for hardware acceleration, load balancers, or ACL functionality within the local network.
 
+(microcloud-networking-underlay)=
+
+Network interface to connect to an OVN underlay network
```

Comment:
Are we talking about a network interface here, or about a network? It sounds like this is a network, unlike the two network interfaces mentioned further up.

If that is true, this formatting is confusing, since it implies that we're talking about a third network interface here.
Instead, turn this into a normal subsection instead of a definition list (in the same way as the internal Ceph network).

If we are talking about a network interface, this one should go into the same list as the other two.

Reply:
fixed